From 80d7fc9b338b28a055ac1d6eb049112baea6755b Mon Sep 17 00:00:00 2001
From: Zineb El Azzouzi <zineb.elazzouzi@nxp.com>
Date: Mon, 14 Aug 2023 17:13:10 +0200
Subject: [PATCH 21/27] arm64: dts: industrial: disable reg_can2_stby

Harpoon application needs GPIO 5 of ADP5585 to set the CAN2_STBY as an
output

Signed-off-by: Zineb El Azzouzi <zineb.elazzouzi@nxp.com>
---
 .../boot/dts/freescale/imx93-11x11-evk-harpoon-industrial.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/imx93-11x11-evk-harpoon-industrial.dts b/arch/arm64/boot/dts/freescale/imx93-11x11-evk-harpoon-industrial.dts
index 31513f7b5a83..d0a18a1dd6dd 100644
--- a/arch/arm64/boot/dts/freescale/imx93-11x11-evk-harpoon-industrial.dts
+++ b/arch/arm64/boot/dts/freescale/imx93-11x11-evk-harpoon-industrial.dts
@@ -13,3 +13,7 @@ &eqos {
 &flexcan2 {
 	status = "disabled";
 };
+
+&reg_can2_stby {
+	status = "disabled";
+};
\ No newline at end of file
-- 
2.34.1

